question-service/controller/questions-controller.js

```diff
@@ -0,0 +1,46 @@
+// Import contact model
+import { ormGetRandomQuestion as _getRandomQuestion, ormGetQuestionByDifficulty as _getQuestionByDifficulty, ormGetQuestionByTopic as _getQuestionByTopic, ormGetTopics as _getTopics } from '../model/questions-orm.js';
+
+
+export async function viewRandomQuestion(req, res) {
+    try {
+        const question = await _getRandomQuestion();
+        return res.status(200).json({ question: question });
+    } catch (err) {
+        res.status(500).json({ message: 'Database failure when getting random question'});
+    }
+};
+
+export async function viewQuestionByDifficulty(req, res) {
+    try {
+        let difficultyLevel = req.params.level;
```

Comment:
We should add a check if difficulty is not provided and return a 400. Else, line 17 will throw an exception. Also, I feel `difficulty` sounds more intuitive than `level` just for your consideration.